components/DocsLayout.js

```diff
@@ -61,6 +61,9 @@ class DocsLayout extends Component {
 
           {children}
         </Content>
+
+        {/* used for integration tests */}
+        <div className="end-of-docs" />
```

Comment:
I'd add a `data-e2e-id` to the root `Content` node instead, otherwise people might remove this eventually and `data-e2e-id` is more explicit :blush: